<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="tracelayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>&lt;status&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="traceView">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="queueFrame">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="1" column="0" colspan="5">
         <widget class="QListWidget" name="queueView">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Queued commands:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>118</width>
               <height>26</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="upButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../../resources/nicos-gui.qrc">
               <normaloff>:/up</normaloff>:/up</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="downButton">
             <property name="icon">
              <iconset resource="../../../../resources/nicos-gui.qrc">
               <normaloff>:/down</normaloff>:/down</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteQueueItem">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearQueue">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="etaWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Estimated Finishing Time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="etaLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>End of time</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionBreak">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/break</normaloff>:/break</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause after current scan point</string>
   </property>
   <property name="toolTip">
    <string>Pause the script after current scan point or command</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/continue</normaloff>:/continue</iconset>
   </property>
   <property name="text">
    <string>&amp;Continue</string>
   </property>
   <property name="toolTip">
    <string>Continue script</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/stop</normaloff>:/stop</iconset>
   </property>
   <property name="text">
    <string>Stop after current scan &amp;point</string>
   </property>
   <property name="toolTip">
    <string>Stop script after current scan point or command</string>
   </property>
  </action>
  <action name="actionEmergencyStop">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/emergency</normaloff>:/emergency</iconset>
   </property>
   <property name="text">
    <string>&amp;Immediate Stop</string>
   </property>
   <property name="toolTip">
    <string>stops script and most devices</string>
   </property>
  </action>
  <action name="actionFinish">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/finish</normaloff>:/finish</iconset>
   </property>
   <property name="text">
    <string>Stop after this &amp;command</string>
   </property>
   <property name="toolTip">
    <string>Finishes the current command and stops the script</string>
   </property>
  </action>
  <action name="actionBreakCount">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/pausecount</normaloff>:/pausecount</iconset>
   </property>
   <property name="text">
    <string>Pause while counting</string>
   </property>
   <property name="toolTip">
    <string>Pause during count, or after current scan point in script</string>
   </property>
  </action>
  <action name="actionFinishEarly">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/finishearly</normaloff>:/finishearly</iconset>
   </property>
   <property name="text">
    <string>&amp;Finish current measurement early and continue</string>
   </property>
   <property name="toolTip">
    <string>Finish current measurement early, continue</string>
   </property>
  </action>
  <action name="actionFinishEarlyAndStop">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/stop-red</normaloff>:/stop-red</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop right now (finish current measurement early)</string>
   </property>
   <property name="toolTip">
    <string>Finish current measurement early and stop</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
